install/src/command.rs

```diff
@@ -1,3 +1,4 @@
+use std::io::RawOsError;
```

Comment:
Can you fold this `use` statement into the larger one beneath it? It should get done by running `./cargo nightly fmt --all` from the top of the repo